approval_frame/views.py

```diff
@@ -36,3 +41,23 @@ def changeUsernameDone(request):
         'registration/username_change_done.html',
         {'new_username': request.user}
     )
+
+
+class CustomRegistrationView(RegistrationView):
+    form_class = RegistrationFormCustom
+
+    def form_valid(self, request, form):
+        form = self.form_class(request.POST)
```

Comment:
I seem to get this error after registering with a valid mailchip_api_key: 
```
  File "/usr/local/lib/python2.7/dist-packages/registration/views.py", line 82, in form_valid
  new_user = self.register(request, **form.cleaned_data)
 AttributeError: 'RegistrationFormCustom' object has no attribute 'cleaned_data'
```

Got to know from [this post](http://stackoverflow.com/questions/8090891/form-object-has-no-attribute-cleaned-data) that perhaps you need to check for form.is_valid() for this attribute to be made.